src/test/groovy/net/rubygrapefruit/platform/file/AbstractFileEventFunctionsTest.groovy

```diff
@@ -419,32 +424,31 @@ abstract class AbstractFileEventFunctionsTest extends Specification {
                 def expectedEventIndex = remainingExpectedEvents.findIndexOf { expectedEvent ->
                     expectedEvent.matches(event)
                 }
+                EventStatus expected
                 if (expectedEventIndex == -1) {
-                    unexpectedEvents << event
+                    expected = UNEXPECTED
```

Comment:
I would have called the variable `status` instead of `expected`.

Reply:
Followed up in https://github.com/gradle/native-platform/pull/204.